DeckBuilder.hpp

```diff
@@ -0,0 +1,15 @@
+#include <string>
+#include <vector>
+
+class DeckBuilder 
+{
```

Comment:
teraz wszystkie pola i funkcje są prywatne. Używaj public i private, jawnie pokazać to, które składowe mają być publiczne a które prywatne